src/components/AboutWhiskey/AboutWhiskey.story.js

```diff
@@ -1,5 +1,13 @@
+<<<<<<< HEAD
```

Comment:
Looks like this Story file still has conflicts.. can you fix? We'll want to save/keep the version without semicolons.